### PR TITLE
process ready items always even if others fail

### PR DESCRIPTION
**Pull request by endevea**
Fixes issue #1218.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         ],
         "linebreak-style": [
             "error",
-            "unix"
+            "windows"
```
Comment (iangilman) [inline .eslintrc.json]: This needs to stay "unix". Is your editor not respecting the linebreak style that's already in the files?

Comment (endevea) [inline .eslintrc.json]: eslint kept complaining. I work on windows. Will set it back to Unix.

**Comment by endevea** [comment]
is there something else I should do now? still a noob

**Comment by iangilman** [comment]
@endevea Nope... just didn't have a chance to get to it earlier. Looking good :) Thank you again for making this fix!

Regarding the eslint thing, you should be able to set your text editor to respect the line break style that's already in the files, regardless of what OS you're on. Which text editor are you using, out of curiosity?

**Comment by endevea** [comment]
VS code is the text editor. However, eslint complained about CRLF/LF in all files when building even before any editing was done. It must be a git setting of mine.

**Comment by iangilman** [comment]
Interesting... yes, sounds like git is somehow converting the line endings!

**Pull request merged**